Detector/DetFCChhHCalTile/src/HCalBarrel_geo.cpp

```diff
@@ -105,10 +105,10 @@ static DD4hep::Geometry::Ref_t createHCal (
   double dy0 = moduleDimensions.dz();
   double dz0 = moduleDimensions.dr() / 2.;
 
-  double drWedge = (dimensions.rmax() - dimensions.rmin()) * 0.5;
+  double drWedge = cos(dphi / 2.) * (dimensions.rmax() - dimensions.rmin()) * 0.5;
```

Comment:
@clementhelsens and @faltovaj - Valentin changed the dimensions of the modules slightly. From the xml description we take the total depth of the calorimeter and create that as a cylindrical envelope. Then we create trapezoids, previously those were slightly extruding from the envelope. This change reduces the sizes slightly (see attachment). Can you comment if we should go with this or rather adjust the envelopes?

![geo_change](https://cloud.githubusercontent.com/assets/5601483/17168894/21346042-53e5-11e6-9834-e08da395ce75.png)